new_game.rb

```diff
@@ -0,0 +1,41 @@
+class NewGame
+    @input
```

Comment:
Instead of using an instance variable, there is also a convention we follow where we use the object initialization pattern:`https://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/objinitialization.html`.  It makes the code more readable as it using set at the first bit.

Reply:
I considered doing that but thought it might be good ux to have an explicit start method the user could call, but if an init method makes more sense I'll change it!

Reply:
Cool - I hav refactored to include an initialize method and a start method - let me know your thoughts :)